### PR TITLE
Update to LC53

### PR DESCRIPTION
**Pull request by janicelmm**
Update to solution LC53

### CODE REVIEW

File: src/Leetcode53.java [hunk 1]
```diff
@@ -1,117 +1,32 @@
...
 public class Leetcode53 {
 	public static int maxSubArray(int[] nums) {
         
-	    if (nums.length == 1)
-	    {
-	        return nums[0];
-	    }
-        
-        if (nums.length == 2)
-	    {
```
Comment (Ewlai) [inline src/Leetcode53.java:11]: formatting

**Review by Ewlai — CHANGES_REQUESTED** [review]
formatting issues

File: src/Leetcode53.java (hunk 1 above)
Comment (janicelmm) [reply src/Leetcode53.java:11]: This line was removed

**Review by janicelmm — COMMENTED** [review]
Please review again.

**Review by Ewlai — CHANGES_REQUESTED** [review]
formatting issues

**Comment by janicelmm** [comment]
fixed

**Review by Ewlai — APPROVED** [review]
LGTM

**Pull request merged**